db/migrate/20171016125613_add_content_title_unique_constraint.rb

```diff
@@ -0,0 +1,19 @@
+class AddContentTitleUniqueConstraint < ActiveRecord::Migration[4.2]
+  def change
+    titles = ForemanOpenscap::ScapContent.unscoped.group(:title).count.select { |key, value| value > 1 }.keys
+    titles.each do |title|
+      duplicates = ForemanOpenscap::ScapContent.unscoped.where :title => title
+      say "#{duplicates.count} Scap Contents with duplicate title detected: #{title}"
+      duplicates.each.with_index do |item, index|
+        next if index == 0
+        new_name = item.title + " #{index + 1}"
+        say "Renaming Scap Content #{item.title} with id #{item.id} to #{new_name}"
+        item.title = new_name
+        item.save!
```

Comment:
This will be prone to errors if in future we add new validations. We can't use the model or we need to skip validations. You can either use `ForemanOpenscap::ScapContent.update` or define fake model.